samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -702,12 +709,20 @@ void manage_connection(void)
 	 * (Once we request connection, the modem will automatically try to reconnect whenever
 	 *  connection is lost).
 	 */
-
-	LOG_INF("Connecting to LTE network. This may take several minutes...");
+	LOG_INF("Setting up LTE...");
 	if (setup_lte()) {
-		LOG_ERR("LTE initialization failed. Continuing anyway. This may fail.");
+		LOG_ERR("LTE setup failed. Continuing anyway.");
 	}
 
+	/* The nRF Cloud library need only be initialized once, and does not need to be reset
+	 * under any circumstances, even error conditions.
+	 */
+	LOG_INF("Setting up nRF Cloud library...");
+	if (setup_cloud()) {
+		LOG_ERR("nRF Cloud library setup failed. Continuing anyway.");
```

Comment:
Is continuing here a good idea?  If the library is not initialized, just about every call into the nrf_cloud lib will fail.
The main cause of init to fail would be memory allocation issues... Perhaps it is best to just print a message and give up or retry... maybe reboot in 5 minutes??

Reply:
I think @plskeggs would object to rebooting.

Since this happens essentially on wake, I also don't think that rebooting would solve any memory allocation issues.

Giving up is an option, I suppose, if this is truly a fatal error. We could consider just giving up for the LTE failure

Reply:
Seems reasonable to me, assuming @plskeggs agrees.

I'll implement the change assuming that he does